<?xml version="1.0"?>
<launch>
  <arg name="object" default="convex" />
  <arg name="grasp_motion_plugin_name" default="whole_body" />

  <node pkg="aerial_transportation" name="aerial_transportation" type="aerial_transportation_node" output="screen">
    <rosparam file="$(find aerial_transportation)/config/aerial_transportation.yaml" command="load" />
    <rosparam file="$(find aerial_transportation)/config/hydrus.yaml" command="load" />

    <param name="searching_file_name" value = "$(find aerial_transportation)/log/searching_result_$(arg object).txt"/>
  </node>

  <!--
  <node pkg="aerial_transportation" name="object_visualization" type="object_visualization" output="screen" if="$(arg ground_truth)">
    <param name = "obj_marker_pub_name" value = "obj_ground_truth_marker" />
    <param name = "uav_state_sub_name" value = "/uav/baselink/odom" />
    <param name = "object_pos_sub_name" value = "/object" />
    <param name = "link_name" value = "link3" />
  </node>
  -->
</launch>
